builder/vsphere/common/step_remote_upload.go

```diff
@@ -42,24 +42,30 @@ func (s *StepRemoteUpload) Run(_ context.Context, state multistep.StateBag) mult
 	return multistep.ActionContinue
 }
 
+func GetRemoteDirectoryAndPath(path string, ds driver.Datastore) (string, string, string, string) {
+	filename := filepath.Base(path)
+	remotePath := fmt.Sprintf("packer_cache/%s", filename)
```

Comment:
I believe remotepath in on the datastore and not on the actual local system running Packer so the the path separator is not dependent on the OS, correct? 

I think it would be nice to just add as documentation to the function that remotepath is platform agnostic because it is on the datastore and not local to the filesystem.

Reply:
that's correct.